docs/platforms/python/tracing/instrumentation/custom-instrumentation/index.mdx

```diff
@@ -69,15 +69,32 @@ def eat_pizza(pizza):
     with sentry_sdk.start_transaction(op="task", name="Eat Pizza"):
         while pizza.slices > 0:
             eat_slice(pizza.slices.pop())
-
 ```
 
-<Alert title="Static & Class Methods" level="warning">
+<Alert title="Static & Class Methods" level="info">
 
 When tracing a static or class method, you **must** add the `@sentry_sdk.trace` decorator **after** the `@staticmethod` or `@classmethod` decorator (i.e., **closer** to the function definition). Otherwise, your function will break!
 
 </Alert>
 
+### Manually
```

Comment:
I think this heading is a bit confusing because the other methods described on this page are also different forms of "manual" instrumentation. Maybe it would be clearer to call this "Manually starting and stopping the span" or something similar

Reply:
done